plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/AppEngineStandardFacet.java

```diff
@@ -56,6 +56,7 @@
   static final String DEFAULT_RUNTIME_NAME = "App Engine Standard";
   public static final String DEFAULT_APPENGINE_SDK_VERSION = "1.9.44";
   public static final String DEFAULT_GCLOUD_PLUGIN_VERSION = "2.0.9.130.v20161013";
+  public static final IProjectFacetVersion GAE_STANDARD = ProjectFacetsManager.getProjectFacet(ID).getVersion(VERSION);
```

Comment:
Let's not make these object final unless there's evidence that they're expensive to compute.